internal/grpc/interceptors/recovery/recovery.go

```diff
@@ -45,8 +45,8 @@ func NewStream() grpc.StreamServerInterceptor {
 }
 
 func recoveryFunc(ctx context.Context, p interface{}) (err error) {
-	stack := debug.Stack()
+	debug.PrintStack()
```

Comment:
Nice!

Reply:
;-) this was needed as I got a full stack printed with literal `\n`'s all over!